### ISSUE TITLE
Remove unnecessary npm modules from the theme package.json

### ISSUE THREAD

**Issue by Narayon** [log-oscon/wp-genesis-boilerplate#53]
There's a lot of modules in the theme `package.json` that are already present in the `package.json` of the starter base.  
This makes update take twice the effort and encourage discrepancies.

**Comment by s3rgiosan** [comment]
Historically and for the purposes of optimizing build times in Codeship, the idea was that there were no dependencies on the starter theme and only exist at the root of the project.

**Comment by Narayon** [comment]
I agree with that approach, but there's still some duplicated packages.  
Moreover, I think the root `package.json` needs a _spring cleaning_.

**Comment by s3rgiosan** [comment]
Agree on that. Keep up the good work 👍
